src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -543,8 +548,9 @@ public static String getTime() {
 	}
 
 	/**
+	 * Gets the metadata for a World

```

Comment:
instead of metadata maybe say status screen

Reply:
Resolved in c27c7e04891a59b2778dd383f2b573495894c26d